src/components/Facility/AssetCreate.tsx

```diff
@@ -180,210 +242,41 @@ const AssetCreate = (props: AssetProps) => {
   );
 
   const assetQuery = useTanStackQueryInstead(routes.getAsset, {
```

Comment:
`useQuery` here.

Reply:
`useTanStackQueryInstead` is also a wrapper around `useQuery` hook right?
anyways made the changes as requested.